dusty/systems/nginx/__init__.py

```diff
@@ -37,5 +20,6 @@ def update_nginx_from_config(nginx_config):
     or tell it to reload its config to pick up what we've
     just written."""
     logging.info('Updating nginx with new Dusty config')
-    _write_nginx_config(nginx_config)
-    _ensure_nginx_running_with_latest_config()
+    temp_path = tempfile.mkstemp()[1]
+    _write_nginx_config(nginx_config, temp_path)
+    sync_local_path_to_vm(temp_path, os.path.join(constants.NGINX_CONFIG_DIR_IN_VM, 'dustyNginx.conf'))
```

Comment:
This does not get included from the main one. This overwrites a location that nginx looks for its config files `/etc/nginx/conf.d`